Tests/ApolloCacheDependentTests/LoadQueryFromStoreTests.swift

```diff
@@ -361,9 +361,9 @@ class LoadQueryFromStoreTests: XCTestCase, CacheTesting {
   }
 
   func testResultContextWithDataFromMixedDates() throws {
-    let yesterday = Date().addingTimeInterval(-.oneDay)
-    let oneHourAgo = Date().addingTimeInterval(-.oneHour)
-    let aYearAgo = Date().addingTimeInterval(-.oneYear)
+    let oneHourAgo = Calendar.current.date(byAdding: .hour, value: -1, to: Date())!
```

Comment:
I would recommend puling `Date()` out at the top here just so these are all guaranteed to be making adjustments to the same date - in theory the `granularity` should help with that but I do feel like only creating one `Date` is probably less error-prone in general

Reply:
Done! This is definitely smarter to do as it would make it less brittle